### PR TITLE
fix(storybook): add storybook back

### PR DESCRIPTION
**Pull request by wtfluckey**
## Goal

* Add storybook
* Remove old stories, add new ones
* Small styling tweaks to doorhanger
* Remove `emotion` from a few more components

storybook

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -71,3 +71,7 @@ Carthage/
 
 # vsCode
 .vscode/
+
+# linaria
+
```
Comment (anthony-liddle) [inline .gitignore:76]: the nittiest of nits, but can we ditch this extra line-break? 🙃

Comment (wtfluckey) [reply .gitignore:76]: Heck yes!

File: src/components/toggle/toggle.stories.js [hunk 2]
```diff
@@ -1,8 +1,12 @@
...
+
+export default {
+  title: 'Components/Toggle',
+  component: Toggle,
+};
+
+export const Active = () => <Toggle active={true} />
+export const Inactive = () => <Toggle active={false} />
+export const Darkmode = () => <Toggle darkMode={true} />
+export const DarkmodeActive = () => <Toggle darkMode={true} active={true} />
```
Comment (anthony-liddle) [inline src/components/toggle/toggle.stories.js:12]: This got me thinking, let's chat at some point about how we want to implement dark mode. I think using a prop might not be how we want to go about it.

**Review by anthony-liddle — APPROVED** [review]

File: src/components/toggle/toggle.stories.js (hunk 2 above)
Comment (wtfluckey) [reply src/components/toggle/toggle.stories.js:12]: Sounds good. I agree that a prop might not be the right approach.

**Pull request merged**